### PR TITLE
cli: change --flavour to an argument

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -70,7 +70,7 @@ Containerized 'Production' environment
 .. code-block:: console
 
     # Initialize environment and cd into <created folder>
-    $ invenio-cli init --flavour=RDM
+    $ invenio-cli init rdm
```
Comment (zzacharo) [inline README.rst:73]: Probably you need to use `RDM` here, right?

Comment (ppanero) [reply README.rst:73]: It has `case_sensitive=False`, so I think `rdm` should work.

Comment (lnielsen) [reply README.rst:73]: I personally prefer a lower-case version in the cli as it's one key less to press (shift). And yes, it's case insensitive.

File: tests/test_cli.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+from invenio_cli.cli import cli
+
+
+@pytest.fixture()
+def runner():
+    """Click CLI runner."""
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        yield runner
```
Comment (fenekku) [inline tests/test_cli.py:24]: This is **very** nice. Thank you for adding that!

**Pull request merged**